classes/person.rb

```diff
@@ -0,0 +1,24 @@
+# Define a base class
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # accessor get methods
+  attr_reader :id
+  attr_accessor :name, :age
+
+  # accessor set methods
+  def of_age
+    @age >= 18
+  end
+
+  private :of_age
+
+  def can_use_services
+    of_age && @parent_permission
+  end
```

Comment:
- [ ] Please make sure that this method is implemented correctly; this method should return `true` if either of the conditions is true (in other words not both of them have to be "true"), as per [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=if%20person%20is%20of%20age%20or%20if%20they%20have%20permission%20from%20parents.) requirement.